docs/Tutorials/Scaling/Kubernetes/deployingk8s-eks.md

```diff
@@ -344,6 +349,49 @@ Where:
         webui-6cc48f4d68-6r8b5                               1/1     Running     0          4m5s
         ```
 
+2. Check that the MPS Certificate is correct in your browser. Go to your FQDN at port 4433.
+
+    ```
+    https://<Your-AWS-FQDN>:4433
+    ```
+
+3. Verify the MPS Certificate in your browser has the correct Issuer information and is Issued to your FQDN.
+
+    ??? warning "Troubleshoot - `Issued to` field showing NaN or blank"
+
+        **If you're certificate is incorrect, we must take a few steps to correct it before continuing.** Otherwise, the AMT device will not be able to connect to the MPS server.
```

Comment:
I might say: 

If your certificate is incorrect, the AMT device will not connect to the MPS server. See Figure 1. 

Follow the steps below to correct the problem.